src/components/Section/Foreign/Activities/BenefitActivity/Benefit.jsx

```diff
@@ -273,13 +273,14 @@ export default class Benefit extends ValidationElement {
           </RadioGroup>
           <Show when={(this.props.BenefitFrequency || {}).value === 'Other'}>
             <div>
-              {i18n.m('foreign.activities.benefit.label.otherBenefit')}
-              <Textarea
+              {/* The 'Other' frequency type has the same fields as the ConinuingBenefit */}
+              <ContinuingBenefit
                 name="OtherBenefit"
```

Comment:
I think we need to keep the `(Provide explanation)` because the paper form has that text there.

This brings up an issue though. Do we need to maintain the textarea that appears before the additional fields? When looking at the paper form, it's unclear if that textarea is necessary. @ryanhofdotgov